plugin/action/throttle/throttle.go

```diff
@@ -55,14 +78,35 @@ type Config struct {
 
 	//> @3@4@5@6
 	//>
-	//> The default events limit that plugin allows per `interval`
+	//> The default events limit that plugin allows per `interval`.
 	DefaultLimit int64 `json:"default_limit" default:"5000"` //*
 
 	//> @3@4@5@6
 	//>
-	//> What we're limiting: number of messages, or total size of the messages
+	//> It defines subject of limiting: number of messages or total size of the messages.
 	LimitKind string `json:"limit_kind" default:"count" options:"count|size"` //*
 
+	//> @3@4@5@6
+	//>
+	//> A kind of limiter: in-memory (limit arrives from config) or redis (current limit arrives from redis and replaces prevoius one).
```

Comment:
`(current limit arrives from redis and replaces prevoius one).` what does it means?